build.cake

```diff
@@ -934,6 +947,7 @@ Task("Install")
 /// </summary>
 Task("All")
     .IsDependentOn("Cleanup")
+    .IsDependentOn("CleanUpMonoAssets")
```

Comment:
`Default` is dependent on `All` and is dependent on `CleanUpMonoAssets`. Will this not result in Mono being installed every time when running default target on a non-Windows OS?

Reply:
this is already the case today, and in some way it makes sense since it's a full rebuild then (as stated in the PR title).
The PR however addresses the quick build only, which also currently reinstalls the assets every time

Reply:
Ah, yes! Makes sense.